CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 0.29.1
+
+BUG FIXES
+
+* SDK
+  * [\#3207](https://github.com/cosmos/cosmos-sdk/issues/3207) - Fix token printing bug
```

Comment:
Printing? Minting?

Reply:
Hehe, I guess they're synonyms, I suppose I just reused the term from physical tokens

Reply:
All good 👍